README.md

```diff
@@ -54,6 +54,10 @@ Behind the scenes, it uses the [Encrypt Query Language](https://github.com/ciphe
 > [!IMPORTANT]
 > **Prerequisites:** Before you start you need to have this software installed:
 >  - [Docker](https://www.docker.com/) — see Docker's [documentation for installing](https://docs.docker.com/get-started/get-docker/)
+> 
+> **Supported architectures:** We currently support `arm64v8` architectures. If you're interested in a `x86_64` Docker image, upvote [this idea](https://github.com/cipherstash/proxy/discussions/214).
```

Comment:
```suggestion
> **Supported architectures:** CipherStash Proxy provides Docker images for these architectures: `linux/arm64`
> If you're interested in a Docker image for other architectures (like `linux/amd64`), upvote [this idea](https://github.com/cipherstash/proxy/discussions/214).
```

Reply:
Also, I think this might be better in a [reference doc](https://github.com/cipherstash/proxy/blob/main/docs/reference.md), rather than in the main README. 

It's too low a level of detail for a getting started guide, but it's the sort of thing that makes sense in a reference doc.